plugins/token-expired.ts

```diff
@@ -12,6 +12,7 @@ const plugin : Plugin = ({ $pinia, app }) => {
     customerStore.setIsLoggedIn(true);
   }
 
+  // eslint-disable-next-line consistent-return
```

Comment:
Why do we disable eslint instead of fixing the issue?

Reply:
Fixed.